src/main/java/io/confluent/connect/elasticsearch/Validator.java

```diff
@@ -117,6 +121,16 @@ private void validateCredentials() {
     }
   }
 
+  private void validateDataStreamFields() {
+    if (config.dataStreamType() == DataStreamType.NONE ^ config.dataStreamDataset() == null) {
+      String errorMessage = String.format(
+          "'%s' must be set if '%s' is set.", DATA_STREAM_DATASET_CONFIG, DATA_STREAM_TYPE_CONFIG
```

Comment:
```suggestion
          "Either both or neither '%s' and '%s' must be set.", DATA_STREAM_DATASET_CONFIG, DATA_STREAM_TYPE_CONFIG
```
let's be consistent with our other validators